### ISSUE TITLE
As by clicking on Export but that will download user have no access to use that image binary data.

### ISSUE THREAD

**Issue by vijaypaytune** [kenvinlu/canva-editor#12]
Export will direct download the Image with Request URL.  if there is way to save binary data to variable. that can be use to preview accordingly and upload that binary data and easily access to the URL anywhere

**Comment by bw-tim** [comment]
Good point, also noticed this in demo. Suggestion would be to add a loader now the export feels to much as a surprise. Optionally a dialog first that shows a preview and give option to provide a filename with an automatic filename suggestion based on the template name plus datetime of export. Having this dialog and the suggested file handling also opens the possibility to do something else with the file first. As in export to other application.

**Comment by duyank** [comment]
@bw-tim @vijaypaytune This project is copy of my project and they are trying to re-sell

Let's use original project to see different (border radius / arrow / SVG / and more features in the future)
https://demo.lidojs.com/
